tests/factories.py

```diff
@@ -71,7 +71,7 @@ def subjects(self, create, extracted, **kwargs):
                 self.subjects.add(subject)
 
     @factory.post_generation
-    def coverage(self, create, extracted, **kwargs):
+    def coverage_locs(self, create, extracted, **kwargs):
```

Comment:
Is this the flake8 issue you mentioned? What was the issue?

Reply:
`coverage` the attribute (line 60) being overridden by `coverage` the method was the issue